src/Apps/Order/Routes/Payment/index.tsx

```diff
@@ -339,13 +342,42 @@ export const PaymentRoute: FC<PaymentRouteProps> = props => {
         selectedPaymentMethod === "SEPA_DEBIT")
     ) {
       handlePaymentStepComplete()
+    } else if (paymentSetupErrorCode && !isPaymentSetupSuccessful) {
```

Comment:
In case there is an error but no error code, we might still want to display the generic error message. Maybe this can just be
```suggestion
    } else if (!isPaymentSetupSuccessful) {
```